docs/getting-started/homepage.md

```diff
@@ -0,0 +1,150 @@
+---
+id: homepage
+title: Backstage Home Page - Setup and Customization
+description: Documentation on setting up and customizing Backstage homepage
+---
+
+## Home Page
+
+Having a good Backstage home page can significantly improve the discoverability of the platform. You want your users to find all the things they need right from the home page and never have to remember direct URLs in Backstage. The [Home plugin](https://github.com/backstage/backstage/tree/master/plugins/home) introduces a system for composing a home page for Backstage in order to surface relevant info and provide convenient shortcuts for common tasks. It's designed with composability in mind with an open ecosystem that allows anyone to contribute with any component, to be included in any home page.
+
+For App Integrators, the system is designed to be composable to give total freedom in designing a Home Page that suits the needs of the organization. From the perspective of a Component Developer who wishes to contribute with building blocks to be included in Home Pages, there's a convenient interface for bundling the different parts and exporting them with both error boundary and lazy loading handled under the surface.
+
+Let's get started by installing the home plugin and creating a simple home page for your Backstage app.
+
+### Setup home page
+
+#### 1. Install the plugin
+
+```
+# From the Backstage repository root
```

Comment:
might be a create-app

Reply:
Indeed, the "repository" word can confusion between create-app and the mono-repo. Replacing it with "Backstage application directory" as we talked about :)